runtests.bat

```diff
@@ -7,7 +7,8 @@ REM SET TestConsole=C:\Program Files (x86)\Microsoft Visual Studio\2017\Comunity
 REM SET TestConsole=C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\Common7\IDE\CommonExtensions\Microsoft\TestWindow\vstest.console.exe
 SET TestConsole=C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow\vstest.console.exe
 SET TestSettings=%PROJECT_ROOT%\Tests\DbgEngTest\WinDbgCs_%Platform%.testsettings
-SET DbgEngTest=%PROJECT_ROOT%\bin\%CONFIGURATION%\Tests\DbgEngTest.dll
+SET DbgEngTest=%PROJECT_ROOT%\bin\\Debug\net461\CsDebugScript.Tests.dll
 
```

Comment:
I used it to run tests multiple times. Verification was that breakpoint tests pass 100 times in the row. I guess it may be useful in the future for similar cases.